cve-2021-4034.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Instead of `#!/bin/bash`, for portability and compability you can use `#!/usr/bin/env sh`.